shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -498,17 +498,19 @@ - (BOOL)attached {
   return _engine != nil;
 }
 
-- (void)updateSemantics:(const FlutterSemanticsUpdate*)update {
+- (void)updateSemantics:(const FlutterSemanticsUpdate2*)update {
   NSAssert(_engine.semanticsEnabled, @"Semantics must be enabled.");
   if (!_engine.semanticsEnabled) {
     return;
   }
-  for (size_t i = 0; i < update->nodes_count; i++) {
-    _bridge->AddFlutterSemanticsNodeUpdate(update->nodes[i]);
+  for (size_t i = 0; i < update->node_count; i++) {
+    const FlutterSemanticsNode2* node = update->nodes[i];
+    _bridge->AddFlutterSemanticsNodeUpdate((*node));
```

Comment:
extra parentheses?

Reply:
Oops not sure how that got there, fixed!